devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/ImageBuild.java

```diff
@@ -46,6 +47,9 @@ public List<Dependency> forcedDependencies() {
 
     @Override
     public Map<String, String> forcedProperties() {
+        if (dryRun) {
```

Comment:
Is there a reason you wouldn't return the forced properties even in a dry run? (so you can verify or otherwise indicate they would be set?)